mysql-test/suite/galera/t/galera_gtid_functions.cnf

```
@@ -0,0 +1,8 @@
+!include ../galera_2nodes_as_master.cnf
```

This makes the test to start with 3 servers, which is not required for this test case. Please change it to 
```suggestion
!include ../galera_2nodes.cnf
```

---

fixed